ballerina/tests/51_client_funtion_utils_negative_cases.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+
+service HelloWorld51 {
+	rpc sendStringBiDi(stream google.protobuf.StringValue) returns (stream google.protobuf.StringValue);
+	rpc sendStringClientStreaming(stream google.protobuf.StringValue) returns (google.protobuf.StringValue);
+	rpc sendStringServerStreaming(google.protobuf.StringValue) returns (stream google.protobuf.StringValue);
+	rpc sendStringUnary(google.protobuf.StringValue) returns (google.protobuf.StringValue);
```

Comment:
If both input and output types are string, wouldn't it be better to have "string" as a prefix in these rpc calls instead of "sendString"

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/333/commits/754dab2544ed50a98bcc3db68be24109232445ed